Editor/EditorCore/EditorToolbarMenuItems.cs

```diff
@@ -53,6 +53,24 @@ static void MenuPerform_NewPolyShapeToggle()
 			}
 		}
 
+		[MenuItem(k_MenuPrefix + "Editors/New Shape Toggle %#k", true, PreferenceKeys.menuEditor + 1)]
+		static bool MenuVerify_NewShapeToggle()
+		{
+			var instance = EditorToolbarLoader.GetInstance<NewShapeToggle>();
+			return instance != null && instance.enabled;
```

Comment:
does probuilder need to be open for this to work? seems like it wouldn't need to, but i'm not sure off-hand if other editor menu items also require this.

Reply:
For now it doesn't, it's the same for polyshape and.... bezier shape.
Moreover I just realized that you can create a bezier shape even if experimental is off through this menu, which does seems weird to me, right?

![pbmenu](https://user-images.githubusercontent.com/66317117/112219967-18dba780-8bfc-11eb-9315-c9ff4427d1d7.png)